manuscript/03.results.md

```diff
@@ -19,7 +19,7 @@ In this section, the interface of the model and possible input/outputs are expla
 The tHMM can provide various statistical outputs as a result of fitting, such as the probability of transition between the states, the most likely state of each individual cell, the most fitted distribution associated with each state, and the most likely number of states in a population (Figure 2c). To address the possibility of incorporating numerous phenotypes that can cause heterogeneity within a population, the model is made flexible in terms of the types of heritable measurements that could be used as clustering features. As shown in figure 2a, any observation can be added into the property list of individual cells. Any new phenotype requires the user to define a possible distribution to which the phenotype belongs.
   
 
-![**The interface of tHMM.** (a) The input data gets converted to the lineage objects with cells instantiated with properties from the data, such as cells' observations, paret-daughter link, generation number, etc. (b) The fitting process includes expectation and maximization steps. (c) Outputs of the model after fitting including the tree of states, the tree of observations, the transition probabilities, and the most likely distribution parameters.](./output/figure2.svg){#fig:2}
+![**The interface of tHMM.** (a) The input data gets converted to the lineage objects with cells instantiated with properties from the data, such as cells' observations, paret-daughter link, generation number, etc. (b) The fitting process includes expectation and maximization steps. (c) Outputs of the model after fitting including the tree of states, the tree of observations, the transition probabilities, and the most likely distribution parameters.](./figures/cartoons/figure2.svg){#fig:2}
```

Comment:
Check that this shows up in the manuscript. If not, ping me and I'll try and fix this.

Reply:
Thank you, will do.

Reply:
They didn't show up @aarmey

Reply:
Ok, I just added a command to copy them to the output folder, so revert this to the original path within the makefile and then they should work.

Reply:
Will do, thank you.